packages/reaction-core/client/templates/products/productGrid/productGrid.html

```diff
@@ -1,4 +1,7 @@
 <template name="productGrid">
+
+  {{> contentFor region="dashboardControls" template="filtrationDashboardControls"}}
```

Comment:
This will need to be removed in v0.12 both `contentFor` and the placement of this.